### DISCUSSION TITLE
来自设备制造商的 HEVC 视频扩展又可以在商店搜到了，但提示无法工作。

### DISCUSSION THREAD

**Discussion by qakcn** [cotaku/Bibi-Release#74] [General]
这个扩展：https://apps.microsoft.com/detail/9n4wgh0z6vhq

又可以在商店中看到了，但是在我的电脑上会有提示**此应用在你的设备上将无法工作**（如图）。

<img width="752" alt="屏幕截图 2024-03-03 192324" src="https://github.com/cotaku/Bibi-Release/assets/1809188/4fe0a2b2-a3e4-42a6-b3a6-55d74a2f1c1e">

**Comment by cotaku** [comment]
这个好像现在随着系统自动安装了，总之我再把链接加回去

**Comment by PandaFiredoge** [comment]
似乎不支持Win11

**Reply by PandaFiredoge** [reply]
可能是Microsoft Store的bug

**Reply by PandaFiredoge** [reply]
这就是Microsoft Store的bug，实际上是可以正常工作的

**Comment by Densionwom** [comment]
![image](https://github.com/user-attachments/assets/eb6d1d1b-f306-4293-958d-49b2be34c910)
我是正常用着，这几天才看不了奈飞4k的，检查了才知道是HEVC不用在这设备上使用，查了几个答案没能解决的，微软的工程师回复都没在点上，插个眼，大佬们有解决方法吗

**Reply by cotaku** [reply]
商店里免费版的一直都不能装，不过可以在网上搜索别人之前缓存过的appx安装，用包名"9n4wgh0z6vhq"或者"Microsoft.HEVCVideoExtension_8wekyb3d8bbwe"应该都能搜到

**Comment by lhyzf** [comment]
可能是需要更新显卡驱动，也不一定需要安装这个拓展
